network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/blog_index_callout_box.html

```diff
@@ -0,0 +1,27 @@
+{% load wagtailcore_tags wagtailimages_tags static i18n %}
+
+<div class="tw-h-full tw-flex tw-flex-col tw-justify-between tw-p-[36px] tw-border tw-border-black tw-shadow-[10px_10px_rgba(0,0,0)]">
```

Comment:
**Suggestion** Since we are reusing this shadow definition over and over, it would be good to make this one of the standard shadow options in the `tailwind.config.js`. 

Also, try to avoid arbitrary values as much as possible. Designs are typically not pixel perfect. Try one of the default spacing values from the spacing scale.

Reply:
updated to use new `tw-popout` class!